groups/main.tf

```diff
@@ -0,0 +1,14 @@
+resource "google_project" "project_x" {
+  name       = "Kevala test group self service"
+  project_id = "kv-test-self-service-groups"
+}
+
+module "kats_cool_group" {
+  source = "./modules/kevala_gcp_groups"
+  group_email = "infra-team-testing@kevala.com"
+  group_members = ["kat.schlorff", "eric.arellano", "hannah.cauldwell", "james.ethridge"]
+  group_name = "kats-cool-group"
+  project_id = google_project.project_x.project_id
+  role_to_grant = "roles/gkehub.admin"
```

Comment:
Note for future us: the likely feature growth I can imagine customers coming to us with is:
* I want this group to have multiple roles
* I want this group to have access to multiple projects
* I want support for custom roles

These features would make sense to spend time on as we consider supporting this kind of infra as a product